packages/client/src/components/BlankWindow/BlankWindow.tsx

```diff
@@ -4,14 +4,8 @@ import cn from 'classnames';
 import { Props } from './types';
 import { ErrorBoundary } from '../ErrorBoundary';
 
-export const BlankWindow = ({
-  children,
-  className
-}: Props): JSX.Element => (
+export const BlankWindow = ({ children, className }: Props): JSX.Element => (
```

Comment:
```suggestion
export const BlankWindow = ({ children, className }: Props) => (
```
Тут не обязательно писать тип TS сам его выведет)